src/login/components/GroupManagement/Groups/EditGroup.js

```diff
@@ -1,14 +1,17 @@
-import React from "react";
+import React, { useState } from "react";
 import InjectIntl from "../../../translation/InjectIntl_HOC_factory";
 import InvitesParent from "../Invites/InvitesParentContainer";
+import DeleteGroup from "./DeleteGroup";
+import EditInvite from "../Invites/EditInvite";
 
 const EditGroup = (props) => {
   const { group } = props;
   const { display_name } = props.group;
+  const [parentId, setNavParent] = useState("invite");
```

Comment:
variable name `parentId` could be confusing because its not actually an id. so I recommend, rename `parentId` to `navParentName` or something else? . what do you think?

Reply:
The variable name refers to the html id attribute of the nav `<li>` that is clicked, so I would not like to use "name" and I think it is ok to call it `parentId`.